get-started/rw-premium-edition-intro.mdx

```diff
@@ -58,6 +58,13 @@ SELECT rw_test_paid_tier();
 
 A result of `t` means the key is valid; an error message indicates an invalid key.
 
+### CLI option for license key
```

Comment:
I think we'd better merge this section into `### Set license key` as an alternative (and automated) method of setting the license key.

Reply:
Updated, please take another look